plugins/MultiSites/MultiSites.php

```diff
@@ -38,6 +38,7 @@ public function addMetricTranslations(&$translations)
             'visits_evolution'    => Piwik::translate('General_ColumnNbVisits') . $appendix,
             'actions_evolution'   => Piwik::translate('General_ColumnNbActions') . $appendix,
             'pageviews_evolution' => Piwik::translate('General_ColumnPageviews') . $appendix,
+            'hits_evolution' => Piwik::translate('General_ColumnHits') . $appendix,
```

Comment:
just formatting, feel free to disregard
```suggestion
            'pageviews_evolution' => Piwik::translate('General_ColumnPageviews') . $appendix,
            'hits_evolution'      => Piwik::translate('General_ColumnHits') . $appendix,
```